hphp/hack/src/utils/pos.ml

```diff
@@ -390,10 +390,12 @@ let set_line_end line pos =
   | _ -> pos
 
 let set_col_end pos_cnum pos =
+  (* temporarily change this function to change both the start and end's columns *)
```

Comment:
Please clean up this temporary code. You could define a separate `set_col_start` function, or even write a helper `empty_pos_from_start` function.